apps/backend/src/services/polling_service.py

```
@@ -5,7 +5,7 @@
 import asyncio
 import json
 import logging
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
```

Ruff (0.12.2)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)